lib/protobuf/nats/config.rb

```diff
@@ -16,6 +16,7 @@ class Config
         :tls_client_key => nil,
         :tls_ca_cert => nil,
         :uses_tls => false,
+        :service_replacements => [],
```

Comment:
Let's just call this `subscription_key_replacements` since that's the name of the method and the string we're modifying.